GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -1068,8 +1068,12 @@ ProcessingResult QtQuickAdapter::processTXDetails(bs::message::SeqId msgId
          }
       }
       else {
-         itTxDet->second->setDetails(txDet); // shouldn't be more than one entry
-         itTxDet->second->setCurBlock(blockNum_);
+         const auto details = itTxDet->second;
+         QMetaObject::invokeMethod(this, [this, details, txDet] {
+            details->setDetails(txDet);
```

Comment:
`details->setDetails()` already has the `invokeMethod` inside - shouldn't it be removed instead?